nncf/experimental/torch/nas/bootstrapNAS/elasticity/elastic_width.py

```diff
@@ -514,7 +530,7 @@ def __init__(self, target_model: NNCFNetwork,
         self._add_dynamic_inputs = add_dynamic_inputs
 
         graph = self._target_model.get_original_graph()
-        prunable_types = [NNCFConv2d.op_func_name, NNCFLinear.op_func_name]
+        prunable_types = [NNCFConv2d.op_func_name, NNCFLinear.op_func_name, NNCFEmbedding.op_func_name]
```

Comment:
"Prunable" types are defined in 2 different places. Potentially, it could lead to inconsistency and increasing burden for developers to make it the same. I'd refactor it by creating `ShapePruningProcessor` in builder and by passing it to handler.

Reply:
Should be resolved in a separate PR
https://github.com/openvinotoolkit/nncf/issues/1326